### DISCUSSION TITLE
"Invisible" Cursor Measurments

### DISCUSSION THREAD

**Discussion by MaMiKoe** [OpenHantek/OpenHantek6022#265] [Ideas]
Due to my post in Q&A of today:
I am referring to the behaviour of openhantek_20211216-31c5ee2-1_win_x64 because I can not test it with Linux (I am a Linux-DAU) for compare:
When setting CH1 or CH2 to OFF "Cursor Measurements" are made (only when "drawing" in the above, "normal" window) and a rectangle is drawn "invisible". So when setting CH1 or CH2 again to "ON" the before "invisible" drawn rectangle becomes visible. If this behaviour does also appear in the Linux-version I would like to suggest to change this: IMHO it would be advantegous, if the visible drawn rectangle and its values would be kept while the channel is set to OFF and drawing an "invisible" rectangle while this OFF-state would only result in displaying the measurements while "invisible" drawing and until the mouse-button is released, and the measurement of the kept "visible" drawn rectangle are displayed again.

**Comment by Ho-Ro** [comment]
Fixed with latest unstable.
